File: curso python/ejemplos python/mayor_2numeros.py
num1 = int (input ('Ingrese el primer número: '))
num2 = int (input ('Ingrese el segundo número: '))

if num1>num2:
    m=num1
if num2>num1:
    m=num2
print (m)

#funcion

def mayor (a,b):
    if a>b:
        may=a
    if b>a:
        may = b
    return may
a=int (input ('ingrese1: '))
b=int (input ('ingrese2:'))
print (f'el mayores:',mayor(a,b))

File: curso c++/ejemplos/mayor_2_numeros.cpp
#include <iostream>
using namespace std;
int main (){
    int num1,num2,m;
    cout << "ingrese el primer numero: ";
    cin >> num1;
    cout << "ingrese el segundo numero: ";
    cin >> num2;
    if (num1> num2)
        m = num1;
    if (num2 > num1)
        m = num2;
    cout << "mayor es: "<<m;
    return 0;
}